builders/interoperability/xcm/core-concepts/sovereign-accounts.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Sovereign Accounts and Reserve-Backed Transfers
+description: Discover how sovereign accounts work on Moonbeam, how to calculate them, and their role in cross-chain asset transfers.
+---
+
+# Overview of Sovereign Accounts
+
+## Introduction {: #introduction }
+
+In Polkadot-based ecosystems, a sovereign account is a special, keyless account controlled by a blockchain’s runtime rather than an individual or organization. These accounts are used to store assets when transferring tokens cross-chain. For example, if you send tokens from a parachain to Moonbeam, the originating parachain locks those tokens in Moonbeam’s sovereign account on the source chain, while a wrapped representation of those tokens is minted on Moonbeam.
```

Comment:
```suggestion
In Polkadot-based ecosystems, a sovereign account is a unique, keyless account controlled by a blockchain’s runtime through XCM rather than an individual or organization. These accounts are used to store assets when transferring tokens cross-chain. For example, if you send a reserve tokens transfer from a parachain to Moonbeam, the originating parachain locks those tokens in Moonbeam’s sovereign account on the source chain, while a wrapped representation of those tokens is minted on Moonbeam.
```